Distribution metrics guide updates (#9804)
DataDog/documentation: content/en/tracing/guide/ddsketch_trace_metrics.md

- Distribution Metrics now power APM Service and Resource pages as Datadog rolls this out for more customers.
- Datadog has migrated all manual queries from dashboards and monitors to the new metrics automatically on your behalf, while retaining the existing metrics to support historical views.

> **I'm using Terraform, what does this change mean for me?**

Reviewer: ```suggestion
**I'm using Terraform. What does this change mean for me?**
```

> - The existing metrics are not going away, this means that your Terraform definitions are still in place and still work well.

Reviewer: ```suggestion
The existing metrics are not going away; your Terraform definitions are still in place and still work.
```

Reviewer 2: @andrewardito how did I miss this? it's not "the existing metrics are not going away". I updated this in my draft to be "The existing metrics are still around"

> - We recommend changing these to use the new DDSketch-based metrics that offer [better precision][4].

Reviewer: ```suggestion
 To take advantage of the [better precision][4] offered by the new DDSketch-based metrics, change your Terraform definitions as shown in the following examples.
```

> - See the below examples for how to update your terraform definitions.

Reviewer: ```suggestion

```
